src/prog_models/prognostics_model.py

```diff
@@ -1264,14 +1305,18 @@ def estimate_params(self, runs : List[tuple] = None, keys : List[str] = None, ti
 
         for i, (times, inputs, outputs) in enumerate(runs):
             has_changed = False
+            if len(times) != len(inputs) or len(inputs) != len(outputs):
+                error = f"Times, inputs, and outputs must be same length for the run at index {i}. Length of times: {len(times)}, Length of inputs: {len(inputs)}, Length of outputs: {len(outputs)}"
+                raise ValueError(error)
+            if len(times) == 0:
+                raise ValueError(f"Times, inputs, and outputs for Run {i} must have at least one element")
             if not isinstance(inputs[0], self.InputContainer):
+                # Error for recent test occurs here.
```

Comment:
This comment should be deleted, but if there is an error occurring, we should deal with this before we finalize the PR